packages/web/src/common/store/pages/audio-rewards/slice.ts

```diff
@@ -128,9 +133,13 @@ const slice = createSlice({
       }>
     ) => {
       const { challengeId, specifiers } = action.payload
-      const userChallenge = state.userChallenges[challengeId]
+
+      const userChallenge = Object.values(
+        state.userChallenges[challengeId] || {}
+      )[0]
```

Comment:
`userChallenge` can be undefined, right?

Reply:
I'm surprirsed TS didn't catch that...